prompttools/experiment/experiments/experiment.py

```diff
@@ -124,7 +124,8 @@ def run(
             for _ in range(runs):
                 self.queue.enqueue(
                     self.completion_fn,
-                    combo,
+                    # We need to filter out defaults that are invalid JSON from the request
+                    {k: v for k, v in combo.items() if v and v != float("inf")},
```

Comment:
1. What the common invalid values for `v`? Where do they come from?
2. Is it possible to have `v = 0` or `v = None` that are valid?
3. If I understand correctly, this merely throws away the `k: v` pair but keeps the rest of the combo?

Reply:
We have some default values that work in python but can't be serialized into JSON and sent over the wire (e.g. `None` or `float("inf")`. We need to toss these out before making the request. The result will be that OpenAI uses the corresponding default for the missing value, so the request should be equivalent to what we track on our end.

Reply:
We should account for `v=0` though, we can keep those.

And yeah, we'll toss any pair that's `None` or `float("inf")` and keep the rest.